libkbfs/folder_block_ops.go

```diff
@@ -557,19 +560,13 @@ func (fbo *folderBlockOps) getFileBlockLocked(ctx context.Context,
 		panic(fmt.Sprintf("Unknown block req type: %d", rtype))
 	}
 
-	// Callers should have already done this check, but it doesn't
-	// hurt to do it again.
-	if !file.isValid() {
```

Comment:
The path doesn't need to be valid here, since `ptr` is passed in.  And for conflict resolution, it won't be valid, so move this check to the caller.